tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -1358,6 +1359,10 @@ void SetBranchesHelper(TTree *inputTree, TTree &outputTree, const std::string &i
          outputBranch->SetAddress(address);
          branchAddress = address;
       }
+      // Set basket size specified by the user, regardless of branch types
+      if(basketSize.has_value()){
+         outputBranch->SetBasketSize(basketSize.value());
+      }
```

Comment:
This code is calling `SetBasketSize` potentially at every event. We want to call this method only once, whenever `outputree.Branch` is called. So this part can be removed.